tests/CMakeLists.txt

```diff
@@ -705,46 +745,60 @@ endif()
 set_tests_properties(ssltest PROPERTIES ENVIRONMENT "srcdir=${TEST_SOURCE_DIR}")
 
 # testdsa
-if(NOT MSVC)
-	add_test(NAME testdsa COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/testdsa.sh)
-else()
-	add_test(NAME testdsa COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/testdsa.bat $<TARGET_FILE:openssl>)
+# testdsa doesn't have an executable instead it uses openssl executable which access various files for IO.
```

Comment:
I think we should group these three tests into a single `if(NOT EMSCRIPTEN)` block then we need this comment only once:

```
# testdsa, testenc and testrsa rely on the openssl executable which accesses various files for IO
# adding these files to --preload-file is infeasible, so we skip these tests.
```

Reply:
should be resolved now